src/functions/public/Auth/Revoke-GitHubToken.ps1

```diff
@@ -0,0 +1,136 @@
+﻿function Revoke-GitHubToken {
```

Comment:
Lets use this for the `Disconnect-GitHub` functions for the supported scenarios.